paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/action/cdc/mysql/MySqlSyncDatabaseAction.java

```diff
@@ -105,6 +110,10 @@ public class MySqlSyncDatabaseAction extends ActionBase {
     @Nullable private final Pattern includingPattern;
```

Comment:
So it should be non-null field? You should modify declarement and invoking place.